libs/sdk-ui-tests-e2e/scripts/run_boiler_app.sh

```diff
@@ -0,0 +1,52 @@
+#!/bin/bash
+# (C) 2021-2023 GoodData Corporation
+
+log() {
+  local now;
+  now=$(date '+%Y-%m-%d %H:%M:%S')
+  echo "$now $*" 1>&2
+}
+
+health_check() {
+    for ((i = 1; i <= 50; i++)); do
+        log "Check Boiler app is up, try $i"
+        if curl -k https://127.0.0.1:8080 ; then
+            return 0
+        else
+            sleep 3
+        fi
+    done
+    return 1
+}
+
+apt-get update && apt-get install -y curl && apt-get install -y jq && apt-get install -y sed && apt-get clean
```

Comment:
can `apt-get update && apt-get install -y curl jq .....` work?

Reply:
it work on my local docker, i'll test with CI run

Reply:
it work on CI too